membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -80,3 +81,19 @@ class AccountController extends LazyLogging {
   def monthlyContributionDetails = paymentDetails[SubscriptionPlan.Contributor, Nothing]
   def digitalPackDetails = paymentDetails[SubscriptionPlan.Digipack, Nothing]
 }
+
+// this is helping us stack future/either/option
+object OptionEither {
+
+  type FutureEither[X] = EitherT[Future, String, X]
+
+  def apply[A](m: Future[\/[String, Option[A]]]): OptionT[FutureEither, A] =
+    OptionT[FutureEither, A](EitherT[Future, String, Option[A]](m))
+
+  def liftOption[A](x: Future[\/[String, A]]): OptionT[FutureEither, A] =
```

Comment:
wasn't sure whether to call all the methods apply, and just let overloading pick the right one - it would look better but I'm wondering if any loss of type safety? @pvighi you might have opinions?